docs/install.md

```diff
@@ -239,6 +238,60 @@ Select the following tabs for the installation description of your target.
     Press the start button (green arrow) to 'boot up' your system based on the DietPi image.
     If you do not have a wired LAN connection you have to change the network settings matching to your environment (files `\boot\dietpi.txt` and `\boot\dietpi-wifi.txt`).
 
+=== "Install in VMware Player"
+
+    <font size="+2">Introduction</font>
+
+    Virtual machine images are great for those occasions where you want to set up a DietPi system very quickly and test things. Also it may be used as a Debian based Linux system with a small footprint for development purposes, e.g. with the X11 window system. The small footprint makes it optimally usable on PCs without a huge built in RAM. Also several VMs may be run for different applications.
+
+    One big advantage of such a VM is that id needs only a couple of minutes coming to a running DietPi system.
```

Comment:
`id` => `it`

reminds me of my:

`bid` => `bit` 😄

Reply:
Ok. Changed (also at VirtualBox).